lib/python/scalgoproto.py

```diff
@@ -129,13 +129,15 @@ def __init__(
         offset: int,
         getter: Callable[["Reader", int, int], B],
         haser: Callable[["Reader", int, int], bool],
+        require_has: bool
```

Comment:
Maybe make this keyword-only, e.g. `def __init__(self, ..., *, require_has: bool)`